extensions/sql-database-projects/src/models/project.ts

```diff
@@ -1367,6 +1482,54 @@ export class Project implements ISqlProject {
 		return firstPropertyElement.childNodes[0].nodeValue!;
 	}
 
+	/**
+	 * Sets the value of project property, overrides value if already present in project file.
+	 * @param propertyName Name of property
+	 * @param propertyValue Value of property
+	 */
+	private async setProjectPropertyValue(propertyName: string, propertyValue: string) {
+		const propertyElements = this.projFileXmlDoc!.documentElement.getElementsByTagName(propertyName);
+
+		let propertyElement: Element | undefined;
+		if (propertyElements.length === 0) {
+			propertyElement = this.addProjectPropertyTag(propertyName);
+		} else {
+			propertyElement = propertyElements[0];
+			if (propertyElement.childNodes.length > 0) {
+				propertyElement.removeChild(propertyElement.childNodes[0]);
+			}
+		}
+
+		propertyElement?.appendChild(this.projFileXmlDoc!.createTextNode(propertyValue));
+		await this.serializeToProjFile(this.projFileXmlDoc);
+	}
+
+	/**
+	 * Adds empty project property tag
+	 * @param propertyTag Tag to add
+	 * @returns Added HTMLElement tag
+	 */
+	private addProjectPropertyTag(propertyTag: string): HTMLElement | undefined {
+		let propertyGroup = this.projFileXmlDoc!.getElementsByTagName(constants.PropertyGroup).item(0);
+		if (propertyGroup === null) {
+			propertyGroup = this.projFileXmlDoc!.createElement(constants.PropertyGroup);
+			this.projFileXmlDoc!.documentElement?.appendChild(propertyGroup);
+		}
+
+		const propertyElement = this.projFileXmlDoc!.createElement(propertyTag);
+		propertyGroup.appendChild(propertyElement);
+		return propertyElement;
+	}
+
+	/**
+	 * Remove project property
+	 * @param propertyTag Tag to remove
+	 */
+	private removeProjectPropertyTag(propertyTag: string) {
+		const propertiesWithTagName = this.projFileXmlDoc!.getElementsByTagName(propertyTag);
```

Comment:
I'd suggest we at least find all property groups and search under those. Just finding all elements with a certain name may remove a lot of non-property related stuff.